basic/src/test/java/org/jboss/hal/testsuite/test/runtime/messaging/XaJmsMetricsTestCase.java

```diff
@@ -0,0 +1,161 @@
+import static org.jboss.hal.testsuite.page.MetricsPage.MONITORED_SERVER;
+import static org.jboss.hal.testsuite.util.ConfigUtils.getDefaultHost;
+
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import org.apache.commons.io.IOUtils;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.jboss.arquillian.drone.api.annotation.Drone;
+import org.jboss.arquillian.graphene.page.Page;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.dmr.ModelNode;
+import org.jboss.hal.testsuite.category.Shared;
+import org.jboss.hal.testsuite.creaper.ManagementClientProvider;
+import org.jboss.hal.testsuite.creaper.ResourceVerifier;
+import org.jboss.hal.testsuite.fragment.ConfigAreaFragment;
+import org.jboss.hal.testsuite.page.runtime.MessagingStatisticsPage;
+import org.jboss.hal.testsuite.test.runtime.deployments.DeploymentsOperations;
+import org.jboss.hal.testsuite.test.runtime.messaging.deployment.MessagingStatisticsBean;
+import org.jboss.hal.testsuite.util.ConfigUtils;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.openqa.selenium.WebDriver;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.wildfly.extras.creaper.core.online.ModelNodeResult;
+import org.wildfly.extras.creaper.core.online.OnlineManagementClient;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+import org.wildfly.extras.creaper.core.online.operations.Values;
+import org.wildfly.extras.creaper.core.online.operations.admin.Administration;
+
+@RunWith(Arquillian.class)
+@Category(Shared.class)
+public class XaJmsMetricsTestCase {
+
+    private static final Logger log = LoggerFactory.getLogger(XaJmsMetricsTestCase.class);
+    private static final OnlineManagementClient client = ManagementClientProvider.createOnlineManagementClient();
+    private static final Administration adminOps = new Administration(client);
+    private static final Operations ops = new Operations(client);
+    private static final DeploymentsOperations deployOps = new DeploymentsOperations(client);
+    private static final String ARCHIVE_NAME = "xa-jms-txn", WAR_FILE_EXTENSION = ".war",
+            GENERATE_STATS_URL = "http://" + ConfigUtils.getUrl().getHost() + ":8080/" + ARCHIVE_NAME,
+            SUBSYSTEM_NAME = "messaging-activemq", CONNECTION_FACTORY_NAME = "activemq-ra", SERVER = "server",
+            DEFAULT = "default", POOLED_CONNECTION_FACTORY = "pooled-connection-factory",
+            STATISTICS_ENABLED = "statistics-enabled";
```

Comment:
I would suggest to put one constant per line, it is more easy to read. Just saying :-)